patches/server/0005-Tuinity-Server-Config.patch

```diff
@@ -153,23 +153,19 @@ index 0000000000000000000000000000000000000000..89a4567a8ee9591bab78c444f45cf733
 +        }
 +
 +        private boolean getBoolean(final String path, final boolean dfl) {
-+            this.config.addDefault(path, Boolean.valueOf(dfl));
-+            return this.config.getBoolean(path, dfl);
++            return this.config.getBoolean(path, TuinityConfig.config.getBoolean(path, dfl));
```

Comment:
world config must have its own section

Reply:
This code will set the global default value, and then use the global value as the default value for each world.